lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -290,6 +290,12 @@ class EngineParagraph implements ui.Paragraph {
     final int end = WordBreaker.nextBreakIndex(_plainText, offset);
     return <int>[start, end];
   }
+
+  @override
+  List<ui.LineMetrics> computeLineMetrics() {
```

Comment:
cc @GaryQian does anything in flutter call this yet? Will returning null be safe here?

Reply:
Nothing calls this yet. API just landed today/yesterday

Reply:
If you look at the overall diff for this PR you'll notice that there's no change here. IOW, returning null is exactly what we're already doing. However, I filed an issue for us to implement this properly: https://github.com/flutter/flutter/issues/39537

Reply:
I updated the TODO with the issue link